share/completions/apm.fish

```diff
@@ -40,7 +36,11 @@ function __fish_apm_using_command
                         return 1
                     end
                 case '*'
-                    return 1
+                    if test $command_seen_once -eq 0
+                        continue
```

Comment:
> That would mean there can't be a package called "stars", "install", "docs", "home" or "open".

> And the "benefit" you mention is that fish won't complete - it's not hooked up to syntax highlighting (that would require using completes "-A" option) and doesn't allow the user to undo the error. It's not really a good experience if I'm mashing the TAB key and all I get is that limp blink once. Or even worse, if it decides to complete files again.

Valid points, I'm new to working with `fish` and even newer to actually doing any contributions with it so I just want to make sure that any code I write/thoughts are getting critiqued so I'd rather be told something is wrong or inefficient early on than spend hours ending up with some amalgamated solution which works but isn't nearly as clean.

> Anyway, this is your completion - I don't use apm, so I don't have a stake in this (apart from wanting fish to be great) or expert knowledge of the tool. If you insist, we can keep it that way.

I wouldn't want anything that doesn't provide a good enough user experience getting merged in anyway since even though they're now my completions the user will just see it as an issue with `fish`! As above, you're a maintainer of the project so your point of view is justifiably one that's important to be noted!